cumulus/zombienet/zombienet-sdk/tests/rpc_collator/rpc_collator_builds_blocks.rs

```diff
@@ -0,0 +1,184 @@
+                    break;
+                }
+                sleep(Duration::from_secs(10)).await;
+            }
+            Err(e) => {
+                log::info!("Error checking for parachain {}: {}", PARA_ID, e);
+                return Err(anyhow!("Error checking for parachain {}: {}", PARA_ID, e));
+            }
+        }
+    }
+    if block_height < 10 {
+        return Err(anyhow!("Parachain {} did not reach block height 10 within 250 seconds", PARA_ID));
+    }
+    log::info!("Parachain {} reached block height 10", PARA_ID);
+
+    // 3. Check that eve reports block height is at least 12 within 250 seconds
+    log::info!("Check that eve reports block height is at least 12 within 250 seconds");
+    let start_time = std::time::Instant::now();
+    let timeout = std::time::Duration::from_secs(250);
+    let mut block_height = 0;
+
+    while start_time.elapsed() < timeout && block_height < 12 {
+        match eve_client
+            .blocks()
+            .at_latest()
+            .await
+        {
+            Ok(block) => {
+                block_height = block.number();
+                log::info!("Eve is at block {}", &block_height);
+                if block_height >= 12 {
+                    break;
+                }
+                sleep(Duration::from_secs(10)).await;
+            }
+            Err(e) => {
+                log::info!("Error checking for parachain {}: {}", PARA_ID, e);
+                return Err(anyhow!("Error checking for parachain {}: {}", PARA_ID, e));
+            }
+        }
+    }
+    if block_height < 12 {
+        return Err(anyhow!("Eve did not reach block height 12 within 250 seconds"));
+    }
+    log::info!("Eve reached block height 12");
+    
+    log::info!("Test successful!");
+    Ok(())
+}
+
+async fn build_network_config() -> Result<NetworkConfig, anyhow::Error> {
+    let images = zombienet_sdk::environment::get_images_from_env();
+    log::info!("Using images: {images:?}");
+    
+    NetworkConfigBuilder::new()
+        .with_relaychain(|r| {
+            r.with_chain("rococo-local")
+                .with_default_command("polkadot")
+                .with_default_image(images.polkadot.as_str())
+                .with_default_args(vec![("-lparachain=debug").into()])
+                .with_node(|node| node.with_name("alice").validator(true))
+                .with_node(|node| node.with_name("bob").validator(true))
+                .with_node(|node| node.with_name("charlie").validator(true))
+                .with_node(|node| node.with_name("one").validator(false))
+                .with_node(|node| node.with_name("two").validator(false))
+                .with_node(|node| node.with_name("three").validator(false))
+        })
+        .with_parachain(|p| {
+            p.with_id(PARA_ID)
+                .cumulus_based(true)
+                .with_default_command("test-parachain")
+                .with_default_image(images.cumulus.as_str())
+                .with_collator(|n| {
```

Comment:
Do we have examples on how to convert the args to the new format ? I have tried like so but it doesnt seem to work (without the comments obv)